spec/clearance/session_spec.rb

```diff
@@ -170,6 +170,31 @@ def stub_status(success)
     end
   end
 
+  context "if same_site is set" do
+    before do
+      Clearance.configuration.same_site = :lax
```

Comment:
Can this just be set to any non-nil value, or is there a collection of options?

Reply:
there's a set of options `nil, :none, :lax, :strict` and a couple of aliases https://github.com/rack/rack/blob/6eda04886e3a57918ca2d6a482fda02a678fef0a/lib/rack/utils.rb#L232-L244,